website/docs/r/cis_waf_group.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM: cis_waf_group"
+sidebar_current: "docs-ibm-resource-cis-waf-group"
+description: |-
+  Provides an IBM Cloud Internet Services WAF Rule Group resource.
+---
+
+# ibm_cis_waf_group
+
+Provides a IBM CIS WAF Rule Group resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to change WAF Groups mode of a domain of a CIS instance
+
```

Comment:
`This is also named as CIS rule set.  Please find CIS rule set tab under WAF of your instance in UI`